angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts

```diff
@@ -207,7 +207,7 @@ export class RecordsListComponent implements OnInit, OnDestroy {
       act: new FormControl((this.queryParams && this.queryParams.act) || null),
       regulation: new FormControl((this.queryParams && this.queryParams.regulation) || null),
       activityType: new FormControl((this.queryParams && this.queryParams.activityType) || null),
-      sourceSystemRef: new FormControl((this.queryParams && this.queryParams.sourceSystemRef) || null),
+      sourceSystemRef: new FormControl((this.queryParams && this.queryParams.sourceSystemRef.split(',')) || null),
```

Comment:
This line fails if sourceSystemRef is null. Probably needs a null check.

Reply:
> This line fails if sourceSystemRef is null. Probably needs a null check.

I'm getting the following on load of the list records page:
![image](https://user-images.githubusercontent.com/51633/84846918-bdddaa00-b004-11ea-805a-8428cbd62863.png)

Reply:
bafd6120d732c2dc459f782faee36eaa19dbd513